Tests/Leb128.lean

```diff
@@ -0,0 +1,138 @@
+import LSpec
+
+import Wasm.Leb128
+
+import YatimaStdLib.Bit
+
+open LSpec
+open Wasm.Leb128
+
+  -- let bigN := 1499559017
+  -- IO.println s!"{uLeb128 bigN} should be 233 232 133 203 5"
+  -- IO.println s!"{reassemble $ npad7 bigN} should be 00000101 11001011 10000101 11101000 11101001"
+  -- IO.println s!"= = = = SIGNED LEB128 TEST = = = ="
+  -- IO.println s!"{sLeb128 (-123456)} should be [192, 187, 120]"
+  -- IO.println s!"{sLeb128 (1)} should be [1]"
+  -- IO.println s!"{sLeb128 (624485)} should be 229, 142, 38"
+
+def lebx : Nat := 624485
+def blebx : ByteArray := ntob lebx
+def bitlebx : List Bit := ByteArray.toBits blebx
+def ulebx : List Bit := unlead bitlebx
+def plebx : List Bit := pad7 ulebx
+def nplebx : List Bit := npad7 lebx
+
+/-
+5> erlang:length( "000010011000011101100101" ).
+24
+8> erlang:length("00001001").
+8
+9> erlang:length("10000111").
+8
+10> erlang:length("01100101").
+8
+11> [ 2#00001001, 2#10000111, 2#01100101 ].
+-/
+def testBlebxIs9_135_101 : TestSeq := Id.run $ do
+  let expected : ByteArray := ByteArray.mk #[9,135,101]
+  let actual := ntob lebx
+  test "Big endian 624485 should be ByteArray.mk #[9,135,101]" $
+    expected.data == actual.data &&
+    actual.data == blebx.data
```

Comment:
I don't understand this line. `blebx := ntob lebx` and `actual := ntob lebx`, so this will be trivially true unless the `ntob` magically changes in the meantime, which is an attack vector that should not be the focus of these tests 😄 What did you mean?

Same question for tests below.